cloudera-integration/csd/descriptor/service.sdl

```diff
@@ -103,6 +103,81 @@
         ]
       }
     },
+    {
+      "name": "LOG_COLLECTOR_KAFKA",
+      "label": "Log Collector Kafka",
+      "pluralLabel": "Log Collectors Kafka",
+      "inExpressWizard": true,
+      "jvmBased" : "true",
+      "parameters": [
+        {
+          "name": "topic",
+          "label": "Kafka topic",
+          "description": "Log collector Kafka Topic",
+          "required": "false",
+          "type": "string",
+          "default": "pulse-kafka-topic"
+        },
+        {
+          "name": "log.collector.max.heap.size",
+          "label": "Log collector max java heap size",
+          "description": "Maximum size for the Java process heap memory. Passed to Java -Xmx. Measured in megabytes.",
+          "type": "memory",
+          "default": 4096,
+          "min": 50,
+          "softMin": 256,
+          "softMax": 16384,
+          "unit": "megabytes",
+          "scaleFactor" : 1.3
+        },
+        {
+          "name": "log.collector.extra.opts",
+          "label": "Additional java options",
+          "description": "These arguments are passed as part of the Java command line.",
+          "type": "string",
+          "default": ""
+        }
+      ],
+      "topology": {
+        "minInstances": "1"
+      },
+      "startRunner": {
+        "program": "scripts/control.sh",
+        "args": [
+          "start_log_collector"
```

Comment:
The start command should be `start_log_collector_kafka`

Reply:
fixed